app-text/scdoc/scdoc-1.5.2.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Standalone tool for generating man pages with a simple syntax"
+HOMEPAGE="https://git.sr.ht/~sircmpwn/scdoc"
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://git.sr.ht/~sircmpwn/scdoc"
+	inherit git-r3
+else
+	SRC_URI="https://git.sr.ht/~sircmpwn/scdoc/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+
+src_prepare() {
+	default
+
+	sed -e 's/-Werror//' \
+		-i Makefile || die 'Failed to patch Makefile'
+}
+
+src_compile() {
+	MY_HS="./scdoc"
+	if tc-is-cross-compiler; then
+		tc-export_build_env
+		MY_HS="./hostscdoc"
+		MAKEOPTS+=" HOST_SCDOC=./hostscdoc"
```

Comment:
Do not *ever* alter MAKEOPTS. Instead, pass extra args to emake.

Reply:
See above, will fix.

Reply:
fixed.